backend/dal/Services/Concrete/ProjectService.cs

```diff
@@ -0,0 +1,285 @@
+
+        #region Methods
+        /// <summary>
+        /// Get an array of project status for the specified 'workflow'.
+        /// </summary>
+        /// <param name="workflow"></param>
+        /// <returns></returns>
+        public IEnumerable<ProjectStatus> GetWorkflow(string workflow)
+        {
+            var status = this.Context.ProjectStatus
+                .AsNoTracking()
+                .OrderBy(s => s.SortOrder)
+                .ThenBy(s => s.Name)
+                .Where(s => EF.Functions.Like(s.Workflow, $"%{workflow}%"));
+
+            return status.ToArray();
+        }
+
+        /// <summary>
+        /// Get the project for the specified 'projectNumber'.
+        /// Will not return sensitive properties unless the user has the `sensitive-view` claim and belongs to the owning agency.
+        /// </summary>
+        /// <param name="id"></param>
+        /// <exception type="KeyNotFoundException">Entity does not exist in the datasource.</exception>
+        /// <returns></returns>
+        public Project Get(string projectNumber)
+        {
+            if (String.IsNullOrWhiteSpace(projectNumber)) throw new ArgumentException("Argument cannot be null, empty or whitespace.", nameof(projectNumber));
+            this.User.ThrowIfNotAuthorized(Permissions.PropertyView);
+
+            // Check if user has the ability to view sensitive properties.
+            var userAgencies = this.User.GetAgencies();
+            var viewSensitive = this.User.HasPermission(Permissions.SensitiveView);
+            var isAdmin = this.User.HasPermission(Permissions.AdminProperties);
+
+            var project = this.Context.Projects
+                .Include(p => p.Status)
+                .Include(p => p.TierLevel)
+                .Include(p => p.Agency)
+                .Include(p => p.Agency.Parent)
+                .Include(p => p.Tasks)
+                .Include(p => p.Tasks).ThenInclude(t => t.Task)
+                .Include(p => p.Properties)
+                .Include(p => p.Properties).ThenInclude(p => p.Parcel)
+                .Include(p => p.Properties).ThenInclude(p => p.Parcel).ThenInclude(p => p.Evaluations)
+                .Include(p => p.Properties).ThenInclude(p => p.Parcel).ThenInclude(p => p.Fiscals)
+                .Include(p => p.Properties).ThenInclude(b => b.Building)
+                .Include(p => p.Properties).ThenInclude(b => b.Building).ThenInclude(b => b.Evaluations)
+                .Include(p => p.Properties).ThenInclude(b => b.Building).ThenInclude(b => b.Fiscals)
+                .AsNoTracking()
+                .FirstOrDefault(p => p.ProjectNumber == projectNumber &&
+                    (isAdmin || userAgencies.Contains(p.AgencyId))) ?? throw new KeyNotFoundException();
+
+            // Remove any sensitive properties from the results if the user is not allowed to view them.
+            if (!viewSensitive)
+            {
+                project?.Properties.RemoveAll(p => p.Parcel.IsSensitive);
```

Comment:
Does it make sense to remove them or should we throw an error?

Reply:
This would only be for the purpose of viewing.  Someone is viewing a project (which they are allowed to do) but they're not allowed to see the sensitive properties.